packages/core/README.md

```diff
@@ -39,17 +39,14 @@ import {RestComponent} from '@loopback/rest';
 import {GrpcComponent} from '@loopback/grpc';
 
 const app = new Application({
-  components: [
-    RestComponent, // REST Server
-    GrpcComponent, // GRPC Server
-  ],
   rest: {
     port: 3000,
   },
   grpc: {
     port: 3001,
   },
 });
+app.components([RestComponent, GrpcComponent]);
```

Comment:
As long as there's a comment here to make it clear (that's why those other lines were commented in that way), I'm cool with this.